rubyforgood/human-essentials
# Add # of disposable diapers purchased and donated to annual report

Closes https://github.com/rubyforgood/human-essentials/issues/4435

### Description
Adds number of diposable diapers purchased _and_ donated to the annual survey report

### Type of change

* New feature (non-breaking change which adds functionality)

### How Has This Been Tested?

1. Log in as org admin
2. Go to annual survey
3. See `Disposable diapers purchased` and `Disposable diapers donated` as part of the report now

### Screenshots
<img width="1920" alt="Screenshot 2024-06-12 at 1 07 28 PM" src="https://github.com/rubyforgood/human-essentials/assets/131324731/6817e0b9-754b-489f-912e-661c92fed2f4">
